File: Recompense.java
public class Recompense {
    private String nom;
    private int bonusXp;

    private Quete quete;

    public Recompense(String nom, int bonusXp) {
        this.nom = nom;
        this.bonusXp = bonusXp;
    }

    public String getNom() { return nom; }
    
    public int getBonusXp() { return bonusXp; }

    public Quete getQuete() { return quete; }
    
    void setQueteInternal(Quete q) { this.quete = q; }
}

File: Classe/Recompense.py
class Recompense:
    def __init__(self, nom: str, bonusXp: int):
        self._nom = nom
        self._bonusXp = bonusXp
        self._quete = None  # type: Quete | None

    def getNom(self) -> str:
        return self._nom

    def getBonusXp(self) -> int:
        return self._bonusXp

    def getQuete(self):
        return self._quete

    # équivalent du "package-private" Java : en Python c'est une convention (underscore)
    def setQueteInternal(self, q) -> None:
        self._quete = q
